purr/templates/single.html

```diff
@@ -0,0 +1,79 @@
+<!-- wp:group {"style":{"spacing":{"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained","wideSize":"1200px"}} -->
+<div class="wp-block-group" style="margin-top:0;margin-bottom:0"><!-- wp:columns {"align":"wide"} -->
+<div class="wp-block-columns alignwide"><!-- wp:column {"width":"25%","style":{"spacing":{"padding":{"right":"var:preset|spacing|30","left":"var:preset|spacing|30"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-column" style="padding-right:var(--wp--preset--spacing--30);padding-left:var(--wp--preset--spacing--30);flex-basis:25%"><!-- wp:group {"style":{"position":{"type":"sticky","top":"0px"},"layout":{"selfStretch":"fixed","flexSize":"25%"},"spacing":{"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained","justifyContent":"center"}} -->
+<div class="wp-block-group" style="margin-top:0;margin-bottom:0"><!-- wp:template-part {"slug":"left-sidebar"} /--></div>
+<!-- /wp:group --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"width":"50%","style":{"spacing":{"padding":{"right":"0","left":"0"}},"border":{"right":{"color":"#f2f2f2","width":"1px"},"top":[],"bottom":[],"left":{"color":"#f2f2f2","width":"1px"}}},"layout":{"type":"constrained"}} -->
```

Comment:
can we use the preset color that we define in theme.json instead of the hex value? then if the user changes the value for the preset, it will change on all instances over the theme